### ISSUE TITLE
Use `@equinor/mad-core`

### ISSUE THREAD

**Issue by dansvend** [equinor/hearing-test#331]
Use `@equinor/mad-core` which will replace the common screens from [equinor/mad-expo-core](https://github.com/equinor/mad-expo-core) like the `LoginScreen`, `ReleaseNoteScrnen`, `SettingsScreen` and `CreateIncidentScreen` as well as custom implementations of `AboutScreen` and setting demo mode.

**Comment by dansvend** [comment]
Blocked by not being able to create ServiceNow incidents in production environment.

**Comment by dansvend** [comment]
@SayyareHamit 

This is ready for testing 👩‍🔬 The login screen, what's new screen, settings screen, about screen, feedback screen, environment banner and service message is now used from `@equinor/mad-core` which features a new design and code.

```
-----------------------------
| Platform     | iOS        |
| Version      | 0.5.0      |
| Build number | 20240109.1 |
| Environment  | Test       |
-----------------------------
```

**Comment by SayyareHamit** [comment]
Works as expected in Test. @dansvend 🐱‍💻

**Comment by dansvend** [comment]
> Works as expected in Test. @dansvend 🐱‍💻

Good news 🥳 Thanks for testing 🙏

**Comment by QiJin-Bouvet** [comment]
Tested with version 0.6.0  Build number 20240213.2
Text in Feedback screen is cropped. This will be fixed in [#426](https://github.com/equinor/mad/issues/426)
